packages/yoga/src/Heading/native/TitleContainer.jsx

```diff
@@ -0,0 +1,68 @@
+import { bool, string } from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+import YogaAvatar from '../../Avatar';
+import Box from '../../Box';
+import Text from '../../Text';
+
+const Container = styled(Box)`
+  margin-top: 20px;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const SubContainer = styled(Box)`
+  justify-content: center;
+`;
+
+const Title = styled(Text.H4)``;
```

Comment:
same here, you don't need to create new components since you not adding any styles